kobo/apps/accounts/templatetags/get_provider_appname.py

```diff
@@ -27,7 +27,9 @@ def get_provider_appname(context, provider=None):
     request = context['request']
     try:
         appname = provider.get_app(request).name
-        return appname
+        if appname:
+            return appname
+        return SocialApp.objects.get_current(provider, request).name
```

Comment:
What is this for?

Reply:
This fixes an issue where setting SSO from environment variables would lead to the provider name not showing up in the frontend.